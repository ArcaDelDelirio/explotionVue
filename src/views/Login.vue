<template>
  <div class="about">
    <NavData />
    <input-data />
  </div>
</template>
<script>
// @ is an alias to /src
import NavData from '@/components/NavData.vue'
import InputData from "../components/inputData";

export default {
  name: 'about',
  components: {
    InputData,
    NavData,
  }
}
</script>