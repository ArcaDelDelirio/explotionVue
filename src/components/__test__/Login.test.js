import { shallowMount } from '@vue/test-utils';
import Login from '../Login';

describe('Login View', () => {
  it('Render the component', () => {
    const wrapper = shallowMount(Login);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
