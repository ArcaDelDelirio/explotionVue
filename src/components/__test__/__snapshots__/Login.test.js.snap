// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login View Render the component 1`] = `
<div class="card-login">
  <form action="">
    <div class="container"><label><b>Username</b></label> <input type="text" placeholder="Enter Username" name="uname" required="required"> <label><b>Password</b></label> <input type="password" placeholder="Enter Password" name="psw" required="required"> <button type="submit">Login</button> <label><input type="checkbox" checked="checked" name="remember"> Remember me
      </label></div>
    <div class="footer-container" style="background-color: rgb(241, 241, 241);"><button type="button" class="cancelbtn">Cancel</button> <span class="psw">
        Forgot
        <a href="#">password?</a></span></div>
  </form>
</div>
`;
